trail/query/coord_conversion.py

```diff
@@ -1,6 +1,7 @@
-import numpy
+import numpy as np
 
-def getXYZFromWcs(self, ra, dec):
+
+def getXYZFromWcs(ra, dec):
```

Comment:
Could live with a docstring, and maybe a better name like `eq2unitsphere` or `equatorial_to_unitsphere` or.... 
Should describe it requires radians.